pkg/util/modules/modules.go

```diff
@@ -131,6 +149,12 @@ func (m *Manager) IsUserVisibleModule(mod string) bool {
 	return false
 }
 
+// IsModuleRegistered returns true if given module was registered
+func (m *Manager) IsModuleRegistered(mod string) bool {
```

Comment:
Let's not introduce new public methods to manager, that are not needed externally.